src/index.js

```diff
@@ -17,6 +25,15 @@ const CardExpiryElement = Element('cardExpiry');
 const CardCVCElement = Element('cardCvc');
 const PostalCodeElement = Element('postalCode');
 
+// IBAN
+const IbanElement = Element('iban', {
+  impliedTokenType: 'bank_account',
+  impliedSourceType: 'sepa_debit',
+});
+
+// iDEAL Bank
+const IdealBankElement = Element('idealBank', {impliedSourceType: 'ideal'});
+
```

Comment:
I'm so happy this involves so little code changing.

Might want to make sure the demo still works after the `impliedSourceType` change goes in.

Reply:
Looks like everything still works!